src/api/stats/getAmmPrices.ts

```diff
@@ -581,6 +581,7 @@ const coinGeckoCoins: Record<string, string[]> = {
   axlusdc: ['axlUSDC'],
   'mimo-parallel-governance-token': ['MIMO'],
   olympus: ['OHM'],
+  firebot: ['FBX'],
```

Comment:
could be sushi lp in oldLpPools.json https://polygonscan.com/address/0xcffbfa978ac3fb10e829ca6b763c307daafe8a77#code

Reply:
made this change